app/src/main/java/com/github/se/travelpouch/ui/dashboard/TravelActivity.kt

```diff
@@ -62,7 +65,7 @@ fun TravelActivitiesScreen(
     activityModelView: ActivityViewModel
 ) {
   val context = LocalContext.current
-
+  val showBanner = remember { mutableStateOf(true) }
```

Comment:
Yes, this seems correct because the value is reseted everytime we launch the application. And once we are on the application, we don't need a remainder every 10 minutes (also we won't spend hours on the app)